### PR TITLE
Updated deb sources location

### PR DESCRIPTION
**Pull request by paulgreenbank**
Changed deb source from "https://apt.kubernetes.io/" to "http://packages.cloud.google.com/apt/" as apt.kubernetes.io generates apt error regarding no Release file preventing install of kube* packages for install.

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md
Comment (divya-mohan0209) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:178]: Small alteration in the link, but otherwise lgtm 

https://packages.cloud.google.com/apt/ kubernetes-xenial main

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Please make the necessary changes.

**Comment by neolit123** [comment]
Please post the exact error, apt version, distro version. Before merging
this change it seem better to first report the problem in the
kubernetes/release repository.

**Review by Arhell — COMMENTED** [review]
Please sign CLA 
https://github.com/kubernetes/website/pull/30651#issuecomment-980488405

**Comment by sftim** [comment]
I haven't reviewed the code (CLA not signed) but with https://github.com/kubernetes/website/pull/30651#issuecomment-980635392 in mind:
/hold

**Comment by jimangel** [comment]
Hey there! @paulgreenbank, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by annajung** [comment]
closing the PR as the author has [not signed the CLA for two weeks](https://kubernetes.io/docs/contribute/participate/pr-wranglers/#when-to-close-pull-requests).
Please feel free to reopen after signing the CLA.

/close

**Pull request closed without merging**
